core/domain/suggestion_services.py

```diff
@@ -528,7 +544,7 @@ def update_position_in_rotation(score_category, user_id):
         score_category, user_id)
 
 
-def check_can_resubmit_suggestion(suggestion_id, user_id):
+def check_can_edit_or_resubmit_suggestion(suggestion_id, user_id):
```

Comment:
This would mean only the author can edit the suggestion right? Make a new function instead to allow both authors as well as reviewers to edit.

Reply:
ok

Reply:
done